Formula/theos.rb

```diff
@@ -0,0 +1,45 @@
+class Theos < Formula
+  desc "Cross-platform tools for developing software for iOS and other platforms"
+  homepage "https://theos.dev"
+  license "GPL-3.0-or-later"
+  head "https://github.com/theos/theos.git", branch: "master"
+
+  stable do
+    url "https://github.com/theos/theos.git",
+      branch:   "master",
+      revision: "e53ec51e92a3716def537c7ff1b5b2a86930700c"
+    version "2.6"
+  end
+
+  keg_only "symlinking Theos makes it unusable"
```

Comment:
This should be reported and fixed upstream

Reply:
Packaging theos is not a priority for the developers it looks like so will have to keep it keg_only for now. Users can add the bin to path, so its still usable

Reply:
Well, Homebrew/core doesn't exist to package everything. If the developers don't want their software packaged we can comply with that.

Reply:
They haven't said it explicitly, I'm guessing based on the issue responses in their repo. But checks are passing now, changed keg_only message

Reply:
You'll have to get them to fix it because this makes the formula a lot less user friendly. With this issue I would not consider the software fit for homebrew-core.

Reply:
Okay, let's keep the pull request open until they fix it then?